src/roulette.py

```diff
@@ -126,3 +126,124 @@ def get(self, bin: int) -> Bin:
         :rtype: Bin
         """
         return self.bins[bin]
+
+
+class BinBuilder:
+    def __init__(self) -> None:
+        pass
+
+    def buildBins(self, wheel: Wheel) -> None:
+        self.create_straight_bets(wheel)
+
+    @staticmethod
+    def create_straight_bets(wheel: Wheel) -> None:
+        odd_for_straight_bet = 35
+        straight_bet_numbers = set(_ for _ in range(37))
+
+        for i in straight_bet_numbers:
+            straight_bet = Outcome(str(i), odd_for_straight_bet)
+            wheel.addOutcome(i, straight_bet)
+        zero_zero_bet = Outcome("00", 35)
+        bin_index_of_zero_zero_outcome = 37
+        wheel.addOutcome(bin_index_of_zero_zero_outcome, zero_zero_bet)
+
+    @staticmethod
+    def create_horizontal_split_bets(wheel: Wheel) -> None:
+        odds_for_split_bet = 17
+
+        first_column_numbers = set()
+        for i in range(12):
+            first_column_numbers.add(i * 3 + 1)
```

Comment:
Should be done using comprehensions. Can check item _6.2 Comprehending Comprehensions_ of Python Tricks book.

Reply:
Done. Thanks.Done. Thanks.